package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@leapfrogtechnology/sync-db",
   "description": "Command line utility to synchronize and version control relational database objects across databases",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

This change includes a new feature. Can you update the minor version instead? @thesamiroli 

```suggestion
  "version": "1.2.0",
```

---

Okay!